merchantapi/src/main/java/fi/hel/verkkokauppa/merchant/api/dto/MerchantDto.java

```diff
@@ -0,0 +1,24 @@
+package fi.hel.verkkokauppa.merchant.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import fi.hel.verkkokauppa.merchant.model.ConfigurationModel;
+import lombok.Data;
+import lombok.ToString;
+
+import javax.validation.constraints.NotBlank;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+@Data
+@ToString
```

Comment:
@Data annotation should contain the @ToString generation also (https://projectlombok.org/features/Data) - no need to specify separate @ToString.